amundsen_application/static/js/components/common/EditableText/index.tsx

```diff
@@ -63,19 +64,16 @@ class EditableText extends React.Component<EditableTextProps, EditableTextState>
 
   componentDidUpdate() {
     const { isDisabled, inEditMode, refreshValue, value } = this.state;
-    if (inEditMode) {
-      autosize(this.textAreaTarget);
-      if (refreshValue && refreshValue !== value && !isDisabled) {
-        // disable the component if a refresh is needed
-        this.setState({ isDisabled: true })
-      }
-      else {
-        // when entering edit mode, place focus in the textarea
-        const textArea = ReactDOM.findDOMNode(this.textAreaTarget);
-        if (textArea) {
-          textArea.focus();
-        }
-      }
+    const textArea = this.textAreaRef.current;
+    if (!inEditMode) return;
```

Comment:
nit: We can switch line 67 and line 68 since no need to get the `textArea` if we're going to end up doing nothing.

Reply:
After learning about temporal dead zones, I've been trying to group my `const` and `let` declarations to the start of each block.